index.bs

```diff
@@ -2408,10 +2408,20 @@ When this method is invoked, the user agent MUST execute the following algorithm
                 <dt id="GetAssn-ConditionalMediation-Interact-FormControl">
                   If <code>|options|.{{CredentialRequestOptions/mediation}}</code> is {{CredentialMediationRequirement/conditional}}
                   and the user interacts with an <{input}> or <{textarea}> form control with an <{input/autocomplete}> attribute whose value
-                  contains a `"webauthn"` [=autofill detail token=],
+                  contains `"webauthn"` as the last [=autofill detail token=],
```

Comment:
Generally happy with this and especially the example, but I'm a little worried that the autofill section of html might update and then this section will become stale. Also, `webauthn` is not always a valid token, even if at the end (e.g. you cannot do `autocomplete="off webauthn"`). How about

```
...<{input/autocomplete}> whose [=non-autofill credential type=] is `"webauthn"`.
```

https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#non-autofill-credential-type

Which essentially delegates all parsing to html.

And then, for the note to make RPs lives easier:

```
Note: The "webauthn" autofill detail token must appear immediately after the last autofill detail token
of type "Normal" or "Contact".

[examples]

These examples are likely to trigger conditional UI across the greatest number of user agents. The expected ordering
of these tokens can be confirmed in the [=autofill detail token=] section of the the WHATWG HTML standard.
```

Reply:
I agree with @nsatragno's suggestions above.

I also don't think the "These examples are likely to trigger..." paragraph is needed, it should be obvious enough that that is the purpose of the examples. I would cut this paragraph to make the size of the note less intrusive.

Reply:
@nsatragno Is `[=non-autofill credential type=]` supposed to link anywhere? Bikeshed doesn't seem to recognize it:

![Screenshot 2023-11-15 at 11 16 26 AM](https://github.com/w3c/webauthn/assets/5166470/b1e364d8-ab9e-4bed-8db8-9d2d71737fb4)

Reply:
uhm I did not try the markup I proposed :S

It should ideally link here: https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#non-autofill-credential-type

Reply:
Thanks for confirming, I figured out how to get it to link 🎉 

![Screenshot 2023-11-15 at 12 09 01 PM](https://github.com/w3c/webauthn/assets/5166470/37a9f04b-7124-4dba-9197-a73258f8f984)

Reply:
I've addressed these changes, marking resolved.